src/components/material/MaterialAvailabilityText/online/MaterialAvailabilityTextOnline.tsx

```diff
@@ -1,27 +1,60 @@
 import * as React from "react";
-import { useGetV1ProductsIdentifier } from "../../../../core/publizon/publizon";
+import {
+  useGetV1LibraryProfile,
+  useGetV1ProductsIdentifier
+} from "../../../../core/publizon/publizon";
 import { useText } from "../../../../core/utils/text";
 import MaterialAvailabilityTextParagraph from "../generic/MaterialAvailabilityTextParagraph";
+import { ManifestationMaterialType } from "../../../../core/utils/types/material-type";
 
 interface MaterialAvailabilityTextOnlineProps {
   isbns: string[];
+  materialType: string;
 }
 
 const MaterialAvailabilityTextOnline: React.FC<
   MaterialAvailabilityTextOnlineProps
-> = ({ isbns }) => {
+> = ({ isbns, materialType }) => {
   const t = useText();
+  const { data: productsData } = useGetV1ProductsIdentifier(isbns[0]);
+  const { data: libraryProfileData } = useGetV1LibraryProfile();
+
+  if (!libraryProfileData || !productsData) return null;
+
   const {
-    data: productsData,
-    isLoading: productsIsLoading,
-    isError: productsIsError
-  } = useGetV1ProductsIdentifier(isbns[0]);
+    maxConcurrentEbookReservationsPerBorrower,
+    maxConcurrentEbookLoansPerBorrower,
+    maxConcurrentAudioReservationsPerBorrower,
+    maxConcurrentAudioLoansPerBorrower
+  } = libraryProfileData;
 
-  if (productsIsLoading || productsIsError || !productsData) return null;
+  let availabilityText = "";
 
-  let availabilityText = t("onlineLimitMonthInfoText", {
-    placeholders: { "@count": "X", "@limit": "Y" }
-  });
+  if (
+    materialType === ManifestationMaterialType.ebook &&
+    maxConcurrentEbookReservationsPerBorrower &&
+    maxConcurrentEbookLoansPerBorrower
+  ) {
+    availabilityText = t("onlineLimitMonthEbookInfoText", {
+      placeholders: {
+        "@count": maxConcurrentEbookReservationsPerBorrower,
```

Comment:
Are we sure this is correct? here we want to show how many ebooks the user has already loaned this month, but according to this variable's name - maxConcurrentEbookReservationsPerBorrower - it is the maximum a singular person can reserve (per month I assume).

Reply:
And I also missed this, but concurrent reservations would mean reservations happening at the same time. So it doesn't really seem to me like it's the monthly quota in either of these variables.